pyspecProcScripts/convert_to_power.py

```diff
@@ -50,31 +65,39 @@ def convert_to_power(filename, exptype, s, fl=None):
             log_array[
                 "time"
             ],  # to the "time" field of the structured array that comes from
-            #      the log
+            #     the log
         )
         .set_units("time", "s")
     )
-    log_vs_time = prscr.dBm2power(log_vs_time)
+    # TODO ☐: these are the power readings on the gigatronics.  They
+    # need to be converted to actual powers by adding in a constant dB
+    # value that represents our directional coupler and the associated
+    # losses.  This number is about (but not exactly) 20.  Alex should
+    # be able to tell you what it is.
+    log_vs_time = prscr.dBm2power(log_vs_time + 20)
```

Comment:
I posted a message about this in the processing scripts channel

Reply:
as I see it, this is the only thing that needs to be taken care of before closing the PR.